<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSignalsWindow</class>
 <widget class="QDialog" name="FilterSignalsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>646</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Фильтрация сигналов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="layoutTimeBoundaries">
     <item>
      <widget class="QLabel" name="labelTimeBoundaries">
       <property name="text">
        <string>Границы по времени, c: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerTimeBoundaries">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxLeftTimeBoundary">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxRightTimeBoundary">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
       </property>
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSignals">
     <property name="title">
      <string>Сигналы</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QListWidget" name="listSignals">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTimeFilters">
     <property name="title">
      <string>Фильтры во временной области</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="layoutOutlier">
        <item>
         <widget class="QCheckBox" name="checkBoxOutlier">
          <property name="text">
           <string>Выбросов: </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerOutlier">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBoxOutlier">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutLinearFilter">
        <item>
         <widget class="QCheckBox" name="checkBoxLinearFilter">
          <property name="text">
           <string>Линейный</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxLeftLinearBoundary">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxRightLinearBoundary">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxBandpassFilter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Фильтр в частотной области</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <layout class="QGridLayout" name="layoutSpectrumSettings">
        <item row="0" column="0">
         <widget class="QLabel" name="labelWeightWindowType">
          <property name="text">
           <string>Тип весового окна:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBoxWeightWindowWidth">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>16384</number>
          </property>
          <property name="value">
           <number>512</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="spaceBandpassFilter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelSmoothWidth">
          <property name="text">
           <string>Ширина сглаживания:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelWeightWindowWidth">
          <property name="text">
           <string>Ширина окна:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelOverlapFactor">
          <property name="text">
           <string>Перекрытие окон:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="spinBoxSmoothWidth">
          <property name="maximum">
           <number>2048</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBoxWeightWindowType">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Хэмминга</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ханна</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Блэкмана</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="spinBoxOverlapFactor">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>0.999999000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutBandpassSettings">
        <item>
         <widget class="QLabel" name="labelBandpassFrequencies">
          <property name="text">
           <string>Границы, Гц</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerBandpassFrequencies">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBoxLowerFrequency">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBoxUpperFrequency">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <double>22.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxInterpolation">
     <property name="title">
      <string>Интерполяция</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="layoutNumberInnerIntervals">
        <item>
         <widget class="QLabel" name="labelNumberInnerIntervals">
          <property name="text">
           <string>Период опроса сигнала: </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerNumberInnerIntervals">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxScanPeriod">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::DefaultStepType</enum>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutInterpolationType">
        <item>
         <widget class="QLabel" name="labelInterpolationType">
          <property name="text">
           <string>Тип: </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spaceInterpolationType">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxInterpolationType">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Линейная</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Сплайн</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutFilter">
     <item>
      <spacer name="spacerFilter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Выполнить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
